locales/pt_BR/pt_BR.json

```diff
@@ -690,24 +690,24 @@
     "You are receiving this email because we received a password reset request for your account.": "Você recebeu esse e-mail porque foi solicitado uma redefinição de senha na sua conta.",
     "You have been invited to join the :team team!": "Foi convidado para se juntar à equipa :team!",
     "You have enabled two factor authentication.": "Você ativou a autenticação em dois fatores.",
     "You have not enabled two factor authentication.": "Você não ativou a autenticação em dois fatores.",
-    "You may accept this invitation by clicking the button below:": "You may accept this invitation by clicking the button below:",
-    "You may cancel your subscription at any time. Once your subscription has been cancelled, you will have the option to resume the subscription until the end of your current billing cycle.": "You may cancel your subscription at any time. Once your subscription has been cancelled, you will have the option to resume the subscription until the end of your current billing cycle.",
+    "You may accept this invitation by clicking the button below:": "Você pode aceitar o convite clicando no botão abaixo:",
+    "You may cancel your subscription at any time. Once your subscription has been cancelled, you will have the option to resume the subscription until the end of your current billing cycle.": "Você pode cancelar sua assinatura a qualquer momento. Assim que sua assinatura for cancelada, você terá a opção de retomá-la até o final do seu ciclo de faturamento atual.",
     "You may delete any of your existing tokens if they are no longer needed.": "Você pode excluir seus tokens existentes se eles não forem mais necessários.",
     "You may not delete your personal team.": "Você não pode excluir o seu time pessoal.",
     "You may not leave a team that you created.": "Você não pode sair de um time que você criou.",
     "Your :invoiceName invoice is now available!": "Your :invoiceName invoice is now available!",
     "Your card was declined. Please contact your card issuer for more information.": "Your card was declined. Please contact your card issuer for more information.",
-    "Your current payment method is :paypal.": "Your current payment method is :paypal.",
-    "Your current payment method is a credit card ending in :lastFour that expires on :expiration.": "Your current payment method is a credit card ending in :lastFour that expires on :expiration.",
+    "Your current payment method is :paypal.": "Sua forma de pagamento atual é :paypal.",
+    "Your current payment method is a credit card ending in :lastFour that expires on :expiration.": "Seu método de pagamento atual é um cartão de crédito com final :lastFour que expira em :expiration.",
     "Your registered VAT Number is :vatNumber.": "Your registered VAT Number is :vatNumber.",
     "Zambia": "Zâmbia",
     "Zimbabwe": "Zimbábue",
-    "Zip \/ Postal Code": "Zip \/ Postal Code",
```

Comment:
Should be :
```
"Zip \/ Postal Code": "CEP \/ Código Postal",
```
Instead of :
```
"Zip / Postal Code": "CEP / Código Postal",
```
Don't remove `\` (Backslash)